Ghidra/Processors/V850E2M/data/languages/Instructions/Special.sinc

```diff
@@ -102,9 +107,10 @@
 }
 
 # NOP - 0000000000000000
+define pcodeop __nop;
 :nop is op0015=0x0
 {
-	PC = inst_next;
+	__nop();
```

Comment:
I would not do this.  I don't know the processor, but any sort of using nope for alignment is going to make for terrible decompiler output

Reply:
Should I leave `PC = inst_next;` or let this Implementation clean?

Reply:
That should be good, I kinda like that.

I have done something like:
```
local tmp = 0;
tmp = tmp;
```
To remove bookmarks about empty implementation

Reply:
> but any sort of using nop for alignment

I am still not familiar with all compilers, but three V850 binaries, that i am tested, uses NOPs only as very small delays in low-level drivers.

BTW, it also applies to Infineon binaries.